src/unix/linux_like/linux/musl/b64/aarch64/align.rs

```diff
@@ -17,9 +17,11 @@ s!{
 
     #[repr(align(16))]
     pub struct mcontext_t {
-        // What we want here is a single [u64; 36 + 512], but splitting things
-        // up allows Debug to be auto-derived.
-        __regs1: [[u64; 18]; 2], // 36
-        __regs2: [[u64; 32]; 16], // 512
+        pub fault_address: ::c_ulong,
+        pub regs: [::c_ulong; 31],
+        pub sp: ::c_ulong,
+        pub pc: ::c_ulong,
+        pub pstate: ::c_ulong,
+        pub __reserved: [u128; 256],
```

Comment:
- We cannot use `u128` here to support old Rust version
- Should be nested to auto-derive
- Should not be `pub` as `__reserved`

Reply:
You are right, updated :)